appveyor.yml

```diff
@@ -52,84 +52,88 @@ build_script:
 
     SET PGPASSWORD=Password12!
 
-    Write-Output "postgresql-x64-10 should start out running"
+    Write-Output "postgresql-x64-12 should start out running"
 
-    Get-Service -Name "postgresql-x64-10"
+    Get-Service -Name "postgresql-x64-12"
 
-    Write-Output "If the binary for postgres in not in 'C:\Program Files\postgresql\10' the rest of this script won't work"
+    Write-Output "If the binary for postgres in not in 'C:\Program Files\postgresql\12' the rest of this script won't work"
 
-    reg query "HKLM\System\CurrentControlSet\Services\postgresql-x64-10" /v "ImagePath"
+    reg query "HKLM\System\CurrentControlSet\Services\postgresql-x64-12" /v "ImagePath"
 
-    Stop-Service postgresql-x64-10
+    Stop-Service postgresql-x64-12
 
-    # postgresql-x64-10 should stop
+    # postgresql-x64-12 should stop
 
-    Get-Service -Name "postgresql-x64-10"
+    Get-Service -Name "postgresql-x64-12"
 
     # we preload timescale, and allow all netowrk connections to postgres
 
-    Add-Content "C:\Program Files\postgresql\10\data\postgresql.conf" "shared_preload_libraries = 'timescaledb'"
+    Add-Content "C:\Program Files\postgresql\12\data\postgresql.conf" "shared_preload_libraries = 'timescaledb'"
 
-    Add-Content "C:\Program Files\postgresql\10\data\postgresql.conf" "log_directory = 'pg_log'"
+    Add-Content "C:\Program Files\postgresql\12\data\postgresql.conf" "logging_collector = on"

```

Comment:
If you add the logging collector line to last in this list, the diff will not looks so strange. Right now it is off by one on each line.

Reply:
Thank you for the comment!  I will pay more attention next time. I put the configuration lines based how they appear in `postgresql.conf`, which might not be so important.